packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -89,6 +91,11 @@ export function InitializeBindings() {
         (bindings: Bindings) => {
           this.bindings = bindings;
 
+          const updateLanguage = () => forceUpdate(this);
```

Comment:
This is the key method, re-rendering the components.
It's equivalent to have a `@Prop` or a `@State` property change and trigger an update of the UI

Reply:
One drawback might be that now, it force update all components when the language changed, versus before it would only update components that had at least one localized string to display, right ?

It's pretty minor, however.

Reply:
I believe it's not a really an issue, the language change won't happen often in our customer interfaces, and almost all of our component use localization. It might also prevent other bugs, like we have seen for the manager, a component that is slotted with others also have to be update so we can see a render.